### DISCUSSION TITLE
怎么设置截屏但是不包含字幕

### DISCUSSION THREAD

**Discussion by slvain27** [hooke007/mpv_PlayKit#519] [Q&A 问答]
怎么设置截屏但是没有字幕,lazy只有一个Ctrl+s,原版的是S和s 一个有字幕一个没字幕

**Comment by hooke007** [comment]
你先看看仓库的input conf再来提问。。。

**Reply by slvain27** [reply]
找到了在那个input_easy里面有说明, 我的原因, 没怎么用过github, 以为仓库的文件和下载的文件内容长得一样,我看看怎么把这个帖子删除了
